cypress/integration/homeWork-tests/calculator.spec.js

```diff
@@ -0,0 +1,57 @@
+import { mainPage } from '../../page_objects/homework-page-values/main-page';
+import { sumWholeNumberValues } from '../../page_objects/homework-page-values/number-values';
```

Comment:
Same file being imported multiple times. Would it make sense to have one import instead? You could export whole class instead of each diff constant in that file

Reply:
Is it necessary to use class? Or maybe it would be enough to use just default export? For example: 'export default {sumWholeNumberValues, sumFractionNumberValue}' etc and then 'import values from' etc? Then access objects with values.sumWholeNumberValues etc?

Reply:
Then u would have "values." In front of every variable. I am enthusiast of not having any cumbersome/repetitive stuff. However these are minor things, just a food for thinking.